script/add-codeql-to-path

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -eu
+
+cd "$(dirname "$0")/.."
+
+gh extensions install github/gh-codeql
```

Comment:
Ideally, I want to check whether `gh codeql` is already installed, and skip this step if so...